extensions/kill-process/src/index.tsx

```diff
@@ -82,8 +92,13 @@ export default function ProcessList() {
   const killProcess = (process: Process) => {
     exec(`kill -9 ${process.id}`);
     setFetchResult(state.filter((p) => p.id !== process.id));
-    clearSearchBar({ forceScrollToTop: true });
```

Comment:
I would also not remove this, or customise it based on this new preference, because after you kill a process, it makes sense to clear the search bar (at least in the majority of cases).

Reply:
+1

Reply:
I've added another preference option to customize the clearSearchBar behavior.